ironfish/src/memPool/memPool.ts

```diff
@@ -115,17 +116,53 @@ export class MemPool {
     await this.feeEstimator.init(this.chain)
   }
 
+  /**
+   *
+   * @returns The number of transactions in the mempool
+   */
   count(): number {
     return this.transactions.size
   }
 
+  /**
+   *
+   * @returns The maximum number of transactions that can be stored in the mempool.
+   *
+   * This assumes that all transactions are the maximum size, and so the true
+   * maximum in practice can be higher.
+   */
+  maxCount(): number {
+    // TODO(holahula): calculating max txn size is taken from ironfish/src/consensus/verifier.ts:260
+    // can the max txn size be retrieved from consensus?
+    const maxTransactionSize =
+      this.chain.consensus.parameters.maxBlockSizeBytes - getBlockWithMinersFeeSize()
+
+    return Math.floor(this.maxSizeBytes() / maxTransactionSize)
+  }
+
   /**
    * @return The number of bytes stored in the mempool
    */
   sizeBytes(): number {
     return this.transactionsBytes
   }
 
+  /**
+   *
+   * @returns The maximum number of bytes that can be stored in the mempool
+   */
+  maxSizeBytes(): number {
```

Comment:
maybe we can just make the `maxBytes` field public. This seems kinda verbose

Reply:
is there a standard between private field + public `get()` function vs public field?

Reply:
don't think we have a standard but less boilerplate is always nice